Constant Google Play Services Crash Error on P6a

GMSCompat is showing this error for Play Services quite frequently. This profile is a secondary user profile.

```
type: crash
osVersion: google/bluejay/bluejay:13/TP1A.221005.003/2022101800:user/release-keys
package: com.google.android.gms:223316044
process: com.google.android.gms.persistent

java.lang.SecurityException: getPackagesForUid: UID 1010204 requires android.permission.INTERACT_ACROSS_USERS_FULL or android.permission.INTERACT_ACROSS_USERS to access user 0.
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3021)
	at android.os.Parcel.createException(Parcel.java:3005)
	at android.os.Parcel.readException(Parcel.java:2981)
	at android.os.Parcel.readException(Parcel.java:2923)
	at android.content.pm.IPackageManager$Stub$Proxy.getPackagesForUid(IPackageManager.java:4789)
	at android.app.ApplicationPackageManager$3.recompute(ApplicationPackageManager.java:1133)
	at android.app.ApplicationPackageManager$3.recompute(ApplicationPackageManager.java:1127)
	at android.app.PropertyInvalidatedCache.query(PropertyInvalidatedCache.java:931)
	at android.app.ApplicationPackageManager.getPackagesForUid(ApplicationPackageManager.java:1146)
	at awzw.k(:com.google.android.gms@223316044@22.33.16 (190400-474895102):4)
	at awzw.d(:com.google.android.gms@223316044@22.33.16 (190400-474895102):2)
	at awzw.c(:com.google.android.gms@223316044@22.33.16 (190400-474895102):0)
	at avib.run(:com.google.android.gms@223316044@22.33.16 (190400-474895102):49)
	at avht.run(:com.google.android.gms@223316044@22.33.16 (190400-474895102):6)
	at zad.c(:com.google.android.gms@223316044@22.33.16 (190400-474895102):6)
	at zad.run(:com.google.android.gms@223316044@22.33.16 (190400-474895102):7)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at zfi.run(:com.google.android.gms@223316044@22.33.16 (190400-474895102):0)
	at java.lang.Thread.run(Thread.java:1012)
	Suppressed: crrw: 
		at tk_trace.gcm-GcmChimeraService_onCreate(Unknown Source:0)
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.pm.ComputerEngine.enforceCrossUserOrProfilePermission(ComputerEngine.java:2960)
	at com.android.server.pm.IPackageManagerBase.getPackagesForUid(IPackageManagerBase.java:655)
	at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:2187)
	at com.android.server.pm.PackageManagerService$IPackageManagerImpl.onTransact(PackageManagerService.java:5943)
	at android.os.Binder.execTransactInternal(Binder.java:1302)


```

--- a/gmscompat_config
+++ b/gmscompat_config
@@ -19,6 +19,9 @@ add_external_experiment_ids false
 [com.google.android.gms.nearby]
 # disable WearableDataListenerService, it needs privileged permissions
 FastPairFeature__enable_wearable_service false
+# use PackageManager#getNameForUid() instead of PackageManager#getPackagesForUid(), the latter needs
+# a shim that is currently absent
+FastPairConfig__sass_get_package_name_from_playback_configuration false
 
 [com.google.android.gms.cast]
 # Prevent crashes in GmsCore caused by calling ConnectivityManager#registerNetworkCallback too many

